Survey/Sources/Presentation/Modules/Home/Views/SurveyItemView.swift

```diff
@@ -68,11 +69,33 @@ struct SurveyItemView: View {
     }
 
     private func mainImageSetup() -> some View {
-        // TODO: Remove dummy cover image url
-        Image(survey.coverImageURL)
-            .resizable()
-            .aspectRatio(contentMode: .fill)
-            .opacity(0.6)
-            .edgesIgnoringSafeArea(.all)
+        AsyncImage(url: survey.largeImageURL, transaction: Transaction(animation: .easeInOut(duration: 1.0))) { phase in
```

Comment:
Optional: I think we can break this line.
```suggestion
        AsyncImage(
        url: survey.largeImageURL, 
        transaction: Transaction(animation: .easeInOut(duration: 1.0))
        ) { phase in
```